src/olympia/search/tests/test_filters.py

```diff
@@ -93,6 +103,19 @@ def test_no_fuzzy_multi_word(self):
         qs_str = json.dumps(qs)
         assert 'fuzzy' not in qs_str
 
+    def test_webextension_boost(self):
+        create_switch('boost-webextensions-in-search')
+
+        # Repeat base test with the switch enabled.
+        qs = self._test_q()
+        functions = qs['query']['function_score']['functions']
+
+        assert len(functions) == 2
+        assert functions[1] == {
+            'weight': WEBEXTENSIONS_WEIGHT,
```

Comment:
Given it's a constant in code and not directly configurable I wonder if we should check for it's actual value.